src/test/kotlin/no/nav/familie/oppdrag/rest/SimuleringControllerIntegrationTest.kt

```diff
@@ -22,6 +23,6 @@ internal class SimuleringControllerIntegrasjonTest {
     @Test
     fun test_etterbetalingsbelop() {
         val response = simuleringController.hentEtterbetalingsbeløp(lagTestUtbetalingsoppdragForFGBMedEttBarn())
-        assertEquals(3162, response.body?.data?.etterbetaling)
+        assertEquals(1054, response.body?.data?.etterbetaling)
```

Comment:
Etter diskusjon. (som oppdatert datagenereringen for testet) så mente han at 1054  er riktig verdi, hvordan 3162 kom inn er et mysterium.